src/devices/Nmea0183/Ais/PayloadDecoder.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Original code taken from https://github.com/yellowfeather/AisParser, under MIT License
```

Comment:
would be good to create an issue in that repo and check if them if they could provide something suitable we could directly re-use or some shared component so that both repos don't maintain it

Reply:
I'll open an issue there, but given the number of bugs I fixed (and the additions I needed), the code is more or less a rewrite, with the exception of the message definitions. The other project seems unmaintened, so I strongly doubt there will be the required support.

Reply:
No reaction so far.